Hazel.UnitTests/StressTests.cs

```diff
@@ -26,5 +30,44 @@ public void StressTestOpeningConnections()
                 connection.Connect(new byte[5]);
             });
         }
+
+        // This was a thing that happened to us a DDoS. Mildly instructional that we straight up ignore it.
+        public void SourceAmpAttack()
+        {
+            var localEp = new IPEndPoint(IPAddress.Any, 11710);
+            var serverEp = new IPEndPoint(IPAddress.Loopback, 11710);
+            using (ThreadLimitedUdpConnectionListener listener = new ThreadLimitedUdpConnectionListener(4, localEp, new ConsoleLogger(true)))
+            {
+                listener.Start();
+
+                Socket socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, ProtocolType.Udp);
+                socket.DontFragment = false;
+
+                try
+                {
+                    const int SIO_UDP_CONNRESET = -1744830452;
```

Comment:
Curious about how this SIO option relates to the DDoS (was trying to look up information). Seemed like if this option is true, the attacker could induce exceptions on this socket by ignoring a response--is this code block just demonstrating that? I see that UdpConnection.cs sets SIO_UDP_CONNRESET to true as well--wouldn't setting this to false be best for avoiding an attack?

Reply:
This option only affects Windows machines. It's an oddity that causes annoying exceptions upon reconnect on Windows, but doesn't exist elsewhere, so setting the option throws on linux/mac.